modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -1,24 +1,34 @@
-:navtitle: Adding a Visual Studio Code Extension
+:navtitle: Adding a Visual Studio Code extension to a workspace
 :keywords: Visual Studio Code extension, user-guide
 
 [id="adding-visual-studio-code-extension_{context}"]
-= Adding a Visual Studio Code Extension
+= Adding a Visual Studio Code extension to a workspace
 
-IDE specific plug-ins are no longer part of devfile with devfile v2. Instead, it uses the specific meta-folder to describe the plug-ins.
+Devfiles in the devfile v2 format don't contain any IDE-specific plug-ins. Instead, the plug-ins are described in a specific meta-folder.
 
-[id="visual-studio-code-extensions-json"]
-== Using `.vscode/extensions.json`
+[id="plug-in-registries"]
+== {prod-short} plug-in registries overview
 
-The easiest way to add a Visual Studio Code extension is to add the extension to the `.vscode/extensions.json` file. Using Che-Theia, the extension will be automatically installed and configured. With Che-Code, the Visual Studio Code fork, at the startup of the workspace, Visual Studio Code shows a recommendation pop-up to install the extension.
+{prod-short} has a default plug-in registry installed in every {prod-short} instance. The Che-Theia IDE searches and installs the extensions from that location. 
```

Comment:
{prod-short} has a default plug-in and extensions registry installed in every {prod-short} instance. The Che-Theia IDE searches and installs the plug-ins and extensions from that location.